Cataclysm/APLs/RogueAssasination.simc

```diff
@@ -16,32 +21,37 @@ actions+=/call_action_list,name=cds
 actions+=/shiv,if=debuff.dispellable_enrage.up
 actions+=/call_action_list,name=defensives
 
-actions.st+=/variable,name=use_filler,value=combo_points.deficit>1|!variable.single_target
+actions.st+=/variable,name=use_filler,value=combo_points.deficit>1|variable.not_pooling||!variable.single_target
 actions.st+=/garrote,if=stealthed.all
-actions.st+=/rupture,if=buff.overkill.up&dot.rupture.remains<tick_time|target.time_to_die>6&dot.rupture.remains<tick_time
+actions.st+=/rupture,if=combo_points>1&buff.overkill.up &refreshable&target.time_to_die-remains>20 | combo_points>1&target.time_to_die > 6 &refreshable
 actions.st+=/berserking,if=buff.bloodlust.up|target.time_to_die<20
 
 actions.st+=/envenom,if=target.health.pct>35&combo_points>4&energy>80&buff.envenom.up|target.health.pct>35&buff.envenom.down&energy>55&combo_points>4
 actions.st+=/fan_of_knives,target_if=!dot.deadly_poison_dot.ticking&(!priority_rotation|dot.garrote.ticking|dot.rupture.ticking),if=variable.use_filler&spell_targets.fan_of_knives>=5
-actions.st+=/fan_of_knives,if=variable.use_filler&(!priority_rotation&spell_targets.fan_of_knives>=5)
-actions.st+=/mutilate,cycle_targets=1,if=!dot.deadly_poison_dot.ticking&(combo_points.deficit>1|!variable.single_target)&spell_targets.fan_of_knives=2
+actions.st+=/fan_of_knives,target_if=!dot.deadly_poison_dot.ticking&(!priority_rotation|dot.garrote.ticking|dot.rupture.ticking),if=variable.use_filler&spell_targets.fan_of_knives>=3
+actions.st+=/mutilate,cycle_targets=1,if=!dot.deadly_poison_dot.ticking&(combo_points.deficit>1|variable.not_pooling|!variable.single_target)&spell_targets.fan_of_knives=2
+
 actions.st+=/mutilate,if=variable.use_filler&((target.health.pct>35)|(target.health.pct<35&!settings.backstab))
 
 actions.st+=/backstab,if=settings.backstab&variable.use_filler&target.health.pct<35
 actions.st+=/envenom,if=target.health.pct<35&combo_points=5&energy>65
 
 actions.cds+=/vendetta,if=cooldown.vandetta.charges=2&target.debuff.garrote.duration>6
-actions.cds+=/cold_blood,if=buff.vendetta.up|target.time_to_die<=120
+actions.cds+=/cold_blood,if=debuff.vendetta.up|target.time_to_die<=120
 actions.cds+=/vendetta,if=target.time_to_die>30&buff.slice_and_dice.up&debuff.rupture.up
 actions.cds+=/vanish,if=energy<50&!buff.stealth.up&!buff.overkill.up
+actions.cds+=/synapse_springs,if=buff.swordguard_embroidery.down&debuff.vendetta.up
+
 actions.cds+=/call_action_list,name=misc_cds
+actions.cds+=/call_action_list,name=items

```

Comment:
Is there an `items` list in here somewhere that I overlooked?

Reply:
it was more of a preperation but was forgoten in there